### PR TITLE
Simplify kubeconform install & get version

### PR DESCRIPTION
**Pull request by nvuillam**

### CODE REVIEW

**Comment by nvuillam** [comment]
## [🦙 MegaLinter](https://megalinter.io/beta) status: ❌ [ERROR](https://github.com/oxsecurity/megalinter/actions/runs/4967998807)

| Descriptor  |                                               Linter                                               |Files|Fixed|                                Errors                                 |Elapsed time|
|-------------|----------------------------------------------------------------------------------------------------|-----|----:|-----------------------------------------------------------------------|------------|
|✅ BASH      |[bash-exec](https://megalinter.io/beta/descriptors/bash_bash_exec)                                  |    6|     |                                                                      0|0.02s       |
|✅ BASH      |[shellcheck](https://megalinter.io/beta/descriptors/bash_shellcheck)                                |    6|     |                                                                      0|0.13s       |
|✅ BASH      |[shfmt](https://megalinter.io/beta/descriptors/bash_shfmt)                                          |    6|    0|                                                                      0|0.41s       |
|✅ COPYPASTE |[jscpd](https://megalinter.io/beta/descriptors/copypaste_jscpd)                                     |yes  |     |no                                                                     |3.13s       |
|✅ DOCKERFILE|[hadolint](https://megalinter.io/beta/descriptors/dockerfile_hadolint)                              |  117|     |                                                                      0|13.89s      |
|✅ JSON      |[eslint-plugin-jsonc](https://megalinter.io/beta/descriptors/json_eslint_plugin_jsonc)              |   23|    0|                                                                      0|2.76s       |
|✅ JSON      |[jsonlint](https://megalinter.io/beta/descriptors/json_jsonlint)                                    |   21|     |                                                                      0|0.19s       |
|✅ JSON      |[v8r](https://megalinter.io/beta/descriptors/json_v8r)                                              |   23|     |                                                                      0|14.33s      |
|⚠️ MARKDOWN  |[markdownlint](https://megalinter.io/beta/descriptors/markdown_markdownlint)                        |  313|    0|[232](https://github.com/oxsecurity/megalinter/actions/runs/4967998807)|121.74s     |
|✅ MARKDOWN  |[markdown-link-check](https://megalinter.io/beta/descriptors/markdown_markdown_link_check)          |  313|     |                                                                      0|5.55s       |
|✅ MARKDOWN  |[markdown-table-formatter](https://megalinter.io/beta/descriptors/markdown_markdown_table_formatter)|  313|    0|                                                                      0|16.92s      |
|✅ OPENAPI   |[spectral](https://megalinter.io/beta/descriptors/openapi_spectral)                                 |    1|     |                                                                      0|1.25s       |
|⚠️ PYTHON    |[bandit](https://megalinter.io/beta/descriptors/python_bandit)                                      |  191|     |[58](https://github.com/oxsecurity/megalinter/actions/runs/4967998807) |2.12s       |
|✅ PYTHON    |[black](https://megalinter.io/beta/descriptors/python_black)                                        |  191|    0|                                                                      0|4.57s       |
|✅ PYTHON    |[flake8](https://megalinter.io/beta/descriptors/python_flake8)                                      |  191|     |                                                                      0|1.83s       |
|✅ PYTHON    |[isort](https://megalinter.io/beta/descriptors/python_isort)                                        |  191|    0|                                                                      0|0.83s       |
|✅ PYTHON    |[mypy](https://megalinter.io/beta/descriptors/python_mypy)                                          |  191|     |                                                                      0|7.07s       |
|✅ PYTHON    |[pylint](https://megalinter.io/beta/descriptors/python_pylint)                                      |  191|     |                                                                      0|11.23s      |
|⚠️ PYTHON    |[pyright](https://megalinter.io/beta/descriptors/python_pyright)                                    |  191|     |[299](https://github.com/oxsecurity/megalinter/actions/runs/4967998807)|19.65s      |
|✅ PYTHON    |[ruff](https://megalinter.io/beta/descriptors/python_ruff)                                          |  191|    0|                                                                      0|0.56s       |
|✅ REPOSITORY|[checkov](https://megalinter.io/beta/descriptors/repository_checkov)                                |yes  |     |no                                                                     |30.55s      |
|✅ REPOSITORY|[git_diff](https://megalinter.io/beta/descriptors/repository_git_diff)                              |yes  |     |no                                                                     |0.39s       |
|✅ REPOSITORY|[secretlint](https://megalinter.io/beta/descriptors/repository_secretlint)                          |yes  |     |no                                                                     |12.48s      |
|✅ REPOSITORY|[trivy](https://megalinter.io/beta/descriptors/repository_trivy)                                    |yes  |     |no                                                                     |26.07s      |
|❌ SPELL     |[cspell](https://megalinter.io/beta/descriptors/spell_cspell)                                       |  619|     |[3](https://github.com/oxsecurity/megalinter/actions/runs/4967998807)  |23.92s      |
|⚠️ SPELL     |[vale](https://megalinter.io/beta/descriptors/spell_vale)                                           |  222|     |[63](https://github.com/oxsecurity/megalinter/actions/runs/4967998807) |27.44s      |
|✅ XML       |[xmllint](https://megalinter.io/beta/descriptors/xml_xmllint)                                       |    3|    0|                                                                      0|0.46s       |
|✅ YAML      |[prettier](https://megalinter.io/beta/descriptors/yaml_prettier)                                    |  157|    0|                                                                      0|4.6s        |
|✅ YAML      |[v8r](https://megalinter.io/beta/descriptors/yaml_v8r)                                              |   99|     |                                                                      0|132.54s     |
|✅ YAML      |[yamllint](https://megalinter.io/beta/descriptors/yaml_yamllint)                                    |  158|     |                                                                      0|1.53s       |

See detailed report in [MegaLinter reports](https://github.com/oxsecurity/megalinter/actions/runs/4967998807)

_MegaLinter is graciously provided by [OX Security](https://www.ox.security/?ref=megalinter)_

**Comment by nvuillam** [comment]
## [🦙 MegaLinter](https://megalinter.io/test-nuillam-kubekonform) status: ⚠️ [WARNING](https://github.com/oxsecurity/megalinter/actions/runs/4968719412)

| Descriptor  |                                                         Linter                                                         |Files|Fixed|                                Errors                                 |Elapsed time|
|-------------|------------------------------------------------------------------------------------------------------------------------|-----|----:|-----------------------------------------------------------------------|------------|
|✅ BASH      |[bash-exec](https://megalinter.io/test-nuillam-kubekonform/descriptors/bash_bash_exec)                                  |    6|     |                                                                      0|0.02s       |
|✅ BASH      |[shellcheck](https://megalinter.io/test-nuillam-kubekonform/descriptors/bash_shellcheck)                                |    6|     |                                                                      0|0.18s       |
|✅ BASH      |[shfmt](https://megalinter.io/test-nuillam-kubekonform/descriptors/bash_shfmt)                                          |    6|    0|                                                                      0|0.04s       |
|✅ COPYPASTE |[jscpd](https://megalinter.io/test-nuillam-kubekonform/descriptors/copypaste_jscpd)                                     |yes  |     |no                                                                     |4.78s       |
|✅ DOCKERFILE|[hadolint](https://megalinter.io/test-nuillam-kubekonform/descriptors/dockerfile_hadolint)                              |  118|     |                                                                      0|23.3s       |
|✅ JSON      |[eslint-plugin-jsonc](https://megalinter.io/test-nuillam-kubekonform/descriptors/json_eslint_plugin_jsonc)              |   23|    0|                                                                      0|3.15s       |
|✅ JSON      |[jsonlint](https://megalinter.io/test-nuillam-kubekonform/descriptors/json_jsonlint)                                    |   21|     |                                                                      0|0.29s       |
|✅ JSON      |[npm-package-json-lint](https://megalinter.io/test-nuillam-kubekonform/descriptors/json_npm_package_json_lint)          |yes  |     |no                                                                     |0.92s       |
|✅ JSON      |[v8r](https://megalinter.io/test-nuillam-kubekonform/descriptors/json_v8r)                                              |   23|     |                                                                      0|18.44s      |
|⚠️ MARKDOWN  |[markdownlint](https://megalinter.io/test-nuillam-kubekonform/descriptors/markdown_markdownlint)                        |  313|    2|[232](https://github.com/oxsecurity/megalinter/actions/runs/4968719412)|182.7s      |
|✅ MARKDOWN  |[markdown-link-check](https://megalinter.io/test-nuillam-kubekonform/descriptors/markdown_markdown_link_check)          |  313|     |                                                                      0|6.5s        |
|✅ MARKDOWN  |[markdown-table-formatter](https://megalinter.io/test-nuillam-kubekonform/descriptors/markdown_markdown_table_formatter)|  313|    2|                                                                      0|22.85s      |
|✅ OPENAPI   |[spectral](https://megalinter.io/test-nuillam-kubekonform/descriptors/openapi_spectral)                                 |    1|     |                                                                      0|1.83s       |
|⚠️ PYTHON    |[bandit](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_bandit)                                      |  192|     |[58](https://github.com/oxsecurity/megalinter/actions/runs/4968719412) |4.58s       |
|✅ PYTHON    |[black](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_black)                                        |  192|    0|                                                                      0|5.46s       |
|✅ PYTHON    |[flake8](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_flake8)                                      |  192|     |                                                                      0|2.64s       |
|✅ PYTHON    |[isort](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_isort)                                        |  192|    0|                                                                      0|0.71s       |
|✅ PYTHON    |[mypy](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_mypy)                                          |  192|     |                                                                      0|11.26s      |
|✅ PYTHON    |[pylint](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_pylint)                                      |  192|     |                                                                      0|17.87s      |
|⚠️ PYTHON    |[pyright](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_pyright)                                    |  192|     |[299](https://github.com/oxsecurity/megalinter/actions/runs/4968719412)|28.62s      |
|✅ PYTHON    |[ruff](https://megalinter.io/test-nuillam-kubekonform/descriptors/python_ruff)                                          |  192|    0|                                                                      0|0.19s       |
|✅ REPOSITORY|[checkov](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_checkov)                                |yes  |     |no                                                                     |43.06s      |
|⚠️ REPOSITORY|[devskim](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_devskim)                                |yes  |     |[913](https://github.com/oxsecurity/megalinter/actions/runs/4968719412)|5.33s       |
|✅ REPOSITORY|[dustilock](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_dustilock)                            |yes  |     |no                                                                     |2.44s       |
|✅ REPOSITORY|[git_diff](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_git_diff)                              |yes  |     |no                                                                     |0.05s       |
|✅ REPOSITORY|[secretlint](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_secretlint)                          |yes  |     |no                                                                     |17.82s      |
|✅ REPOSITORY|[syft](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_syft)                                      |yes  |     |no                                                                     |1.4s        |
|✅ REPOSITORY|[trivy](https://megalinter.io/test-nuillam-kubekonform/descriptors/repository_trivy)                                    |yes  |     |no                                                                     |32.71s      |
|✅ SPELL     |[cspell](https://megalinter.io/test-nuillam-kubekonform/descriptors/spell_cspell)                                       |  634|     |                                                                      0|38.89s      |
|⚠️ SPELL     |[vale](https://megalinter.io/test-nuillam-kubekonform/descriptors/spell_vale)                                           |  222|     |[63](https://github.com/oxsecurity/megalinter/actions/runs/4968719412) |37.61s      |
|✅ XML       |[xmllint](https://megalinter.io/test-nuillam-kubekonform/descriptors/xml_xmllint)                                       |    3|    0|                                                                      0|0.03s       |
|✅ YAML      |[prettier](https://megalinter.io/test-nuillam-kubekonform/descriptors/yaml_prettier)                                    |  157|    0|                                                                      0|5.01s       |
|✅ YAML      |[v8r](https://megalinter.io/test-nuillam-kubekonform/descriptors/yaml_v8r)                                              |   99|     |                                                                      0|168.29s     |
|✅ YAML      |[yamllint](https://megalinter.io/test-nuillam-kubekonform/descriptors/yaml_yamllint)                                    |  158|     |                                                                      0|2.17s       |

See detailed report in [MegaLinter reports](https://github.com/oxsecurity/megalinter/actions/runs/4968719412)

_MegaLinter is graciously provided by [OX Security](https://www.ox.security/?ref=megalinter)_

File: megalinter/descriptors/kubernetes.megalinter-descriptor.yml [hunk 1]
```diff
@@ -53,23 +56,14 @@ linters:
...
-        - |
-          RUN ML_THIRD_PARTY_DIR="/third-party/kubeconform" \
-              && KUBECONFORM_VERSION=v0.5.0 \
-              && mkdir -p ${ML_THIRD_PARTY_DIR} \
-              && wget -P ${ML_THIRD_PARTY_DIR} -q https://github.com/yannh/kubeconform/releases/download/$KUBECONFORM_VERSION/kubeconform-linux-amd64.tar.gz \
-              && tar xf ${ML_THIRD_PARTY_DIR}/kubeconform-linux-amd64.tar.gz --directory ${ML_THIRD_PARTY_DIR} \
-              && mv ${ML_THIRD_PARTY_DIR}/kubeconform /usr/local/bin \
-              && rm ${ML_THIRD_PARTY_DIR}/kubeconform-linux-amd64.tar.gz \
-              && find ${ML_THIRD_PARTY_DIR} -type f -not -name 'LICENSE*' -delete -o -type d -empty -delete
+        - RUN go install github.com/yannh/kubeconform/cmd/kubeconform@latest
```
Comment (echoix) [inline megalinter/descriptors/kubernetes.megalinter-descriptor.yml:66]: @nvuillam does this linter need the go compiler for its runtime? If not, this linter could be installed with a stage like I already changed once for dustilock or revive, by using a stage to install. 

https://github.com/oxsecurity/megalinter/blob/46b3721de31e2a1b5f0416512191f3d2b5f93b1f/megalinter/descriptors/go.megalinter-descriptor.yml#L77-L88

https://github.com/oxsecurity/megalinter/blob/46b3721de31e2a1b5f0416512191f3d2b5f93b1f/megalinter/descriptors/repository.megalinter-descriptor.yml#L123-L130

**Review by echoix — COMMENTED** [review]
Installing with go install, means we have to install the go compiler, and we have the build artifacts in the image.

**Comment by nvuillam** [comment]
@echoix look here ;) https://github.com/oxsecurity/megalinter/pull/2637

**Comment by echoix** [comment]
Ok, thanks! I didn't finish going through all the changes during the last day yet!

**Comment by nvuillam** [comment]
> Ok, thanks! I didn't finish going through all the changes during the last day yet!

I decided we'll be going to a v7 ... there are too many changes, some of them potentially breaking changes, so you are the team are right, let's respect SEMVER even if it's just by precaution :)

**Pull request merged**